File: exercism/python/pangram/pangram.py
import re

# returns True if sentence is pangram
def is_pangram(sentence):
    
    # s = set()
    # for e in sentence:
    #     s.add(e.lower())
    # for x in range(ord('a'), ord('z')+1):
    #     if chr(x) not in s:
    #         return False
    # return True
    return len(set(re.sub('[^a-z]', '', sentence.lower()))) == 26

File: exercism/go/pangram/pangram.go
package pangram

import (
	"regexp"
	"strings"
)

// IsPangram returns is input is a pangram
func IsPangram(input string) bool {

	set := make(map[rune]bool)

	r, _ := regexp.Compile("[a-z]")

	for _, ch := range strings.ToLower(input) {
		if r.MatchString(string(ch)) {
			set[ch] = true
		}
	}

	return len(set) == 26
}
